src/components/Header/HeaderMenuBmrg.js

```diff
@@ -8,15 +8,14 @@ const { prefix } = settings;
 const HeaderMenuBmrg = React.forwardRef((props, ref) => {
   const { isOpen, ...other } = props;
   return (
-    <div
+    <button
```

Comment:
Made this a proper button and fixed some styling for it to match better w/ the rest of the header